### PR TITLE
apps/userdashboard/notifications: making alerts appearing as design a…

### PR DESCRIPTION
**Pull request by khamui**
…nd with optional timer

### CODE REVIEW

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-
 import { ModerationStatement } from './ModerationStatement'
+import Alert from '../../../../contrib/assets/Alert'
+
+const translated = {
+  statementAdded: 'Your note was successfully delivered.',
+  statementEdited: 'Your note was successfully updated.',
+  statementDeleted: 'Your note was successfully deleted.',
+  goToDiscussion: 'Go to discussion'
+}
```
Comment (Rineee) [inline apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:14]: translations! :) so we need ```django.pgettext('kosmo', 'string')``` here

Comment (Rineee) [inline apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:14]: translations! :) so we need ```django.pgettext('kosmo', 'string')``` here

Comment (khamui) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:14]: oops

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx [hunk 2]
```diff
@@ -82,7 +82,7 @@ export default class ModerationNotification extends Component {
           showModeratorStatementForm: false,
           alert: {
             type: 'success',
-            message: translated.statementAdded,
+            message: (() => { return (<>{translated.statementAdded} <a href={commentUrl}>{translated.goToDiscussion}</a></>) })(),
```
Comment (goapunk) [inline apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:85]: @khamui I couldn't really come up with a nice solution without changing the alert component. Maybe that's too hacky?

Comment (khamui) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:85]: hehe, well at least it works. I would keep the hacky solution and comment it in code?
later we can adjust the alert to make those kind of things (and maybe other things that is needed) work as well..? If you cool with that, feel free to merge @goapunk

**Review by khamui — COMMENTED** [review]
nice! thanks for adding it

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx (hunk 2 above)
Comment (khamui) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:85]: btw i think if you use `=> function` you can remove `{}` and `return`

Comment (goapunk) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx]: I turned it into a real function for now and added a comment

**Comment by khamui** [comment]
cool! working fine

**Pull request merged**